quickstart.sh

```diff
@@ -88,30 +88,53 @@ echo -e "${MAJOR}Waiting for OpenSearch to start up and be online.${RESET}"
 ./opensearch/wait-for-os.sh # Wait for OpenSearch to be online
 
 echo -e "${MAJOR}Creating ecommerce-keyword index, defining its mapping & settings\n${RESET}"
-curl -s -X PUT "http://localhost:9200/ecommerce-keyword/" -H 'Content-Type: application/json' --data-binary @./opensearch/schema.json
+curl -s -X PUT "http://localhost:9201/ecommerce-keyword/" -H 'Content-Type: application/json' --data-binary @./opensearch/schema.json
 echo -e "\n"
 
 echo -e "${MAJOR}Creating ecommerce alias for ecommerce-keyword index\n${RESET}"
-curl -s -X POST "http://localhost:9200/ecommerce-keyword/_aliases/ecommerce" -H "Content-Type: application/json"
+curl -s -X POST "http://localhost:9201/ecommerce-keyword/_aliases/ecommerce" -H "Content-Type: application/json"
 echo -e "\n"
 
 echo -e "${MAJOR}Prepping Data for Ingestion\n${RESET}"
-if [ ! -f ./icecat-products-w_price-19k-20201127.tar.gz ]; then
+if [ ! -f ./esci.json.zst ]; then
   echo -e "${MINOR}Downloading the sample product data\n${RESET}"
-  wget http://querqy.org/datasets/icecat/icecat-products-w_price-19k-20201127.tar.gz
+  wget https://esci-s.s3.amazonaws.com/esci.json.zst
 fi
 
-if [ ! -f ./icecat-products-w_price-19k-20201127.json ]; then
+if [ ! -f ./esci.json ]; then
   echo -e "${MINOR}Unpacking the sample product data, please give it a few minutes!\n${RESET}"
-  tar xzf icecat-products-w_price-19k-20201127.tar.gz
+  zstd --decompress esci.json.zst 
+  # create a sample for starters
+  head -n 10000 esci.json > esci_10000.json
 fi
 
 echo -e "${MINOR}Transforming the sample product data into JSON format, please give it a few minutes!\n${RESET}"
-docker run -v ./:/app -w /app python:3 python3 ./opensearch/transform_data.py icecat-products-w_price-19k-20201127.json transformed_data.json
-
-echo -e "${MAJOR}Indexing the sample product data, please wait...\n${RESET}"
```

Comment:
should we keep this echo?

Reply:
good catch, added
´´´
echo -e "${MAJOR}Indexing the product data, please wait...\n${RESET}"
´´´
above the indexing part of the script